src/example-types/nonzeroClusters.js

```diff
@@ -25,7 +25,7 @@ function calcClusters(entries, interval, isSignificant) {
 }
 
 module.exports = {
-  validContext: context => context.config.field,
+  validContext: context => context.field,
   result: (context, search) =>
     histogramResult(context, search).then(histResult => ({
```

Comment:
this is dependent on `histogramResult` being modified isn't it?

Reply:
test pass without that 🤔

Reply:
I mean, the tests for nonzeroClusters